src/sql/parts/objectExplorer/viewlet/serverTreeActionProvider.ts

```diff
@@ -27,8 +27,15 @@ import { ConnectionProfile } from 'sql/parts/connection/common/connectionProfile
 import { NewProfilerAction } from 'sql/parts/profiler/contrib/profilerActions';
 import { TreeUpdateUtils } from 'sql/parts/objectExplorer/viewlet/treeUpdateUtils';
 import { IConnectionManagementService } from 'sql/parts/connection/common/connectionManagement';
-import { ExecuteCommandAction } from 'vs/platform/actions/common/actions';
+import { MenuId, IMenuService } from 'vs/platform/actions/common/actions';
 import { NewQueryAction } from 'sql/workbench/common/actions';
+import { ConnectionContextkey } from 'sql/parts/connection/common/connectionContextKey';
```

Comment:
seems like this class should be named `ConnectionContextKey` (with `K`)..though doesn't look like you're adding this class in this PR

Reply:
Yep, I can fix this however. Will do it now since you recommended it.